lib/packages.ts

```diff
@@ -164,7 +164,7 @@ function _getVersionFromGit(experimental: boolean): string {
   }
 
   const hasLocalChanges = _exec(`git status --porcelain`) != '';
-  const scmVersionTagRaw = _exec(`git describe --match v[0-9]*.[0-9]*.[0-9]* --abbrev=7 --tags`)
+  const scmVersionTagRaw = _exec(`git describe --match v[0-9]*.[0-9]*.[0-9]* --abbrev=7 --tags --always`)
```

Comment:
This doesn't seem right, because `--always` will return commit SHA and not a tag version. 

I think the original problem is that tags were not fetched or are not in sync with upstream.

Reply:
I will update the commit message. 

FYI, this was a fresh build (on a new computer) and followed the installation guide.